### PR TITLE
Autonomous and miscellaneous debugging

### PR DESCRIPTION
**Pull request by celskeggs**

### CODE REVIEW

File: ZukoAzula2016/src/org/team1540/zukoazula/ZukoAzula.java [hunk 1]
```diff
@@ -15,6 +15,7 @@
     @Override
     public void setupRobot() throws ExtendedMotorFailureException {
         Logger.info("🐣 CHEEP CHEEP 🐣");
+        //Instrumentation.setup();
```
Comment (AidanSmith) [inline ZukoAzula2016/src/org/team1540/zukoazula/ZukoAzula.java:18]: Reason for this?

File: ZukoAzula2016/src/org/team1540/zukoazula/DriveCode.java [hunk 2]
```diff
@@ -39,6 +39,7 @@
     private static final Behavior teleop = behaviors.addBehavior("Teleop", FRC.inTeleopMode());
     private static final BooleanCell pitModeEnable = new BooleanCell();
     private static final Behavior pit = behaviors.addBehavior("Pit Mode", pitModeEnable.andNot(FRC.isOnFMS()));
+    private static final Behavior challenge = behaviors.addBehavior("Challenge Brake", ChallengeBrake.driveBrake);
```
Comment (AidanSmith) [inline ZukoAzula2016/src/org/team1540/zukoazula/DriveCode.java:42]: This behavior acts exactly the same as pit mode. Is there a reason to have them be different?

**Comment by AidanSmith** [comment]
I added comments.

File: ZukoAzula2016/src/org/team1540/zukoazula/ZukoAzula.java (hunk 1 above)
Comment (celskeggs) [inline ZukoAzula2016/src/org/team1540/zukoazula/ZukoAzula.java:18]: Uncomment it if you want it to record the next time the robot starts up.

File: ZukoAzula2016/src/org/team1540/zukoazula/AutonomousModeLowBar.java [hunk 3]
```diff
@@ -5,7 +5,7 @@
 
 public class AutonomousModeLowBar extends AutonomousBase {
 
-    @Tunable(5.0f)
+    @Tunable(18.0f)
```
Comment (AidanSmith) [inline ZukoAzula2016/src/org/team1540/zukoazula/AutonomousModeLowBar.java:8]: Will this have to be tuned again after we fix the encoder?

File: ZukoAzula2016/src/org/team1540/zukoazula/DriveCode.java (hunk 2 above)
Comment (celskeggs) [inline ZukoAzula2016/src/org/team1540/zukoazula/DriveCode.java:42]: Clarity to the driver. The behavior arbitrator can publish an RConf view for the PoultryInspector, which would specify which mode is active.

File: ZukoAzula2016/src/org/team1540/zukoazula/ZukoAzula.java (hunk 1 above)
Comment (AidanSmith) [inline ZukoAzula2016/src/org/team1540/zukoazula/ZukoAzula.java:18]: Oh, got it. :+1:

File: ZukoAzula2016/src/org/team1540/zukoazula/AutonomousModeLowBar.java (hunk 3 above)
Comment (celskeggs) [inline ZukoAzula2016/src/org/team1540/zukoazula/AutonomousModeLowBar.java:8]: Maybe. Or maybe just halved? Or maybe the new tick count of 100 just needs to be doubled.

**Comment by AidanSmith** [comment]
Alright, then it all looks good.

**Pull request merged**